src/popup/App/Home/OperationsList/OperationsList.js

```diff
@@ -0,0 +1,96 @@
+/* global chrome */
```

Comment:
Because we don't have TypeScript here we are required to tell the code that `chrome` variable does exist globally. However this file is not using the global variable so it makes the linter fail. I've removed this line.

I also added a new script on package.json, you can check for lint errors with `yarn lint`